homeassistant/components/smarthab/cover.py

```diff
@@ -20,12 +20,20 @@
 SCAN_INTERVAL = timedelta(seconds=60)
 
 
-def setup_platform(hass, config, add_entities, discovery_info=None):
-    """Set up the SmartHab roller shutters platform."""
+async def async_setup_entry(hass, config_entry, async_add_entities):
+    """Set up SmartHab covers from a config entry."""
+    hub = hass.data[DOMAIN][DATA_HUB]
+    await _async_setup(hass, hub, async_add_entities)
+
 
+async def async_setup_platform(hass, config, async_add_entities, discovery_info=None):
```

Comment:
You can get rid of all the `async_setup_platform` since the config will be imported